lib/router.dart

```diff
@@ -48,5 +52,14 @@ class AppRouter extends _$AppRouter {
             AutoRoute(page: AccountRoute.page, path: 'account'),
           ],
         ),
+        ModalBottomSheetAutoRoute(
+          page: SortRoute.page,
+          path: '/sort',
+          shape: const RoundedRectangleBorder(
+            borderRadius: BorderRadius.vertical(
+              top: Radius.circular(FlexSizes.borderRadiusLg),
+            ),
+          ),
```

Comment:
Strange to have UI declared here, is there a way to include that in the modal definition?